### PR TITLE
Lemma 6.1.2.

### PR DESCRIPTION
**Pull request by mariainesdff**
We formalize the proof of Lemma 6.1.2 and fix some typos in the blueprint.
Note that the new constant in the lemma could be changed to $$2^{108a^3}$$ if preferred.

### CODE REVIEW

**Comment by fpvandoorn** [comment]
Thanks!

Given that $$a \le 4$$ ( http://florisvandoorn.com/carleson/docs/Carleson/Defs.html#PreProofData.four_le_a ) can you improve the bound to $$2 ^ {107a^3}$$ (or better)? 
I'd prefer to have the simpler bound in the form $$2 ^ {Ca^3}$$

Also, can you merge master?

**Comment by mariainesdff** [comment]
> Given that 4 ≤ a ( http://florisvandoorn.com/carleson/docs/Carleson/Defs.html#PreProofData.four_le_a ) can you improve the bound to 2 107 a 3 (or better)? I'd prefer to have the simpler bound in the form 2 C a 3
> 
> Also, can you merge master?

Done!

**Comment by mariainesdff** [comment]
I am not sure what is wrong with the blueprint. It seems fine in the master branch, and I have not modified it in this commit...

File: blueprint/src/chapter/main.tex [hunk 1]
```diff
@@ -3281,11 +3281,12 @@ \section{The density arguments}\label{sec-TT*-T*T}
     Let $x\in X$.
     Then
     \begin{equation}\label{hlmbound}
-    | \sum_{\fp \in \mathfrak{A}}T_{\fp} f(x)|\le 2^{107 a^3} M_{\mathcal{B}} f (x) \, .
+    | \sum_{\fp \in \mathfrak{A}}T_{\fp} f(x)|\le 2^{107 a^3 + a} M_{\mathcal{B}} f (x) \, .
```
[the remark this answers is not in the document]
Comment (mariainesdff) [reply blueprint/src/chapter/main.tex]: Done.

File: blueprint/src/chapter/main.tex [hunk 2]
```diff
@@ -3316,13 +3317,17 @@ \section{The density arguments}\label{sec-TT*-T*T}
 and the doubling property \eqref{doublingx} $5 +100a^2$ times estimates
 the last display by
 \begin{equation}
-\le \frac{2^{5a+101a^3}}{\mu(B(x, 8D^{\ps(\fp)}))}\, .
+\le \frac{2^{5a+101a^3}}{\mu(B(x, 8D^{\ps(\fp)}))}\,
+\end{equation}
+which, thanks to the closeness of the points $x$ and $\pc(\fp)$ shown in \eqref{eqtttt0}, is in turn bounded by
+\begin{equation}
+\le \frac{2^{6a+101a^3}}{\mu(B(\pc(\fp), 8D^{\ps(\fp)}))}\.
```
[the remark this answers is not in the document]
Comment (mariainesdff) [reply blueprint/src/chapter/main.tex]: Hmm possibly. I guess the error was already then in last week's commit.

**Pull request merged**